Unittest MockDataExports::test_remove_excess_exports fails when using S3 Storage.

The test tries to verify if exported file is cleaned-up properly. 
Even if S3, this condition could be verified. Boto can be used to test if a file exists.

Returned error is:
```
def path(self, name):
        """
            Returns a local filesystem path where the file can be retrieved using
            Python's built-in open() function. Storage systems that can't be
            accessed using open() should *not* implement this method.
            """
>       raise NotImplementedError("This backend doesn't support absolute paths.")
E       NotImplementedError: This backend doesn't support absolute paths.
```

--- a/kpi/tests/test_mock_data_exports.py
+++ b/kpi/tests/test_mock_data_exports.py
@@ -345,6 +345,7 @@ class MockDataExports(TestCase):
         export_task.run()
         self.assertEqual(export_task.status, ExportTask.COMPLETE)
         result = export_task.result
+        self.assertTrue(result.storage.exists(result.name))
         # Make an excessive amount of additional exports
         excess_count = 5 + settings.MAXIMUM_EXPORTS_PER_USER_PER_FORM
         for _ in range(excess_count):
@@ -364,7 +365,7 @@ class MockDataExports(TestCase):
         export_task.run()
         self.assertEqual(export_task.status, ExportTask.COMPLETE)
         # Verify the cleanup
-        self.assertFalse(result.storage.exists(result.path))
+        self.assertFalse(result.storage.exists(result.name))
         self.assertListEqual( # assertSequenceEqual isn't working...
             list(export_tasks_to_keep.values_list('pk', flat=True)),
             list(ExportTask._filter_by_source_kludge(

